majordodo-core/src/main/java/majordodo/replication/ReplicatedCommitLog.java

```diff
@@ -561,16 +569,29 @@ private void openNewLedger() throws LogNotAvailableException {
             writer = new CommitFileWriter();
             currentLedgerId = writer.getLedgerId();
             LOGGER.log(Level.INFO, "Opened new ledger:" + currentLedgerId);
-            if (actualLedgersList.getFirstLedger() < 0) {
```

Comment:
it looks like you are dropping these line
why ?

Reply:
Because I noticed that in actualLedgersList:

    public void addLedger(long id) {
        activeLedgers.add(id);
        ledgersTimestamps.add(System.currentTimeMillis());
        if (firstLedger < 0) {
            firstLedger = id;
        }
    }

Reply:
good